spec/models/npq_application_spec.rb

```diff
@@ -80,6 +80,34 @@
       end
     end
 
+    context "when transient_previously_funded is declared on the model" do
```

Comment:
We should have one test for `#eligible_for_dfe_funding` and `#ineligible_for_funding_reason` each where by default it does not respond_to? `transient_previously_funded`

Reply:
That case is already covered by the existing tests?